src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java

```diff
@@ -18,7 +17,7 @@
         "TEST_URL");
 
```

Comment:
Can we add a tests to cover service authorisation usage?

i.e  Test showing success with service authorisation. Forbidden examples are ok as they depict what happens when no service authorisation is provided via header